projects/components/src/confirmation-modal/confirmation-modal.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ConfirmationModalComponent } from './confirmation-modal.component';
+
+// TODO: Add tests here
```

Comment:
Assuming these are still wip since they're not using spectator.

Reply:
Yep. PR is still a work in progress. Tests, cleanup, and reorganizing pending.